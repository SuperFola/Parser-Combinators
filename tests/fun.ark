(fun () 1)
(
fun
(

)


"12"
)
(fun(a b) 2)
(
fun
(
cc
dddd
)
1
)