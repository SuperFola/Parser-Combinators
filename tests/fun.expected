( ( Keyword:fun ( ) Number:1 ) ( Keyword:fun ( ) String:12 ) ( Keyword:fun ( Symbol:a Symbol:b ) Number:2 ) ( Keyword:fun ( Symbol:cc Symbol:dddd ) Number:1 ) )